packages/tree-view/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { FrameworkConfiguration, PLATFORM } from 'aurelia-framework';
+import { UxDefaultTreeViewConfiguration } from './ux-default-tree-view-configuration';
+import { UxTreeView } from './ux-tree-view/ux-tree-view';
+
+export function configure(config: FrameworkConfiguration, callback?: (config: UxDefaultTreeViewConfiguration) => void) {
+  config.globalResources([
+    PLATFORM.moduleName('./ux-tree-view/ux-tree-view'),
+    PLATFORM.moduleName('./ux-tree-node/ux-tree-node')
```

Comment:
is `ux-tree-node` supposed to be a global custom element? if not we can move this line to `<require/>` in tree view, as it seems currently we are only using it as a pseudo custom element

Reply:
done